README.md

```diff
@@ -72,3 +72,9 @@ To modify download infomation or change packaging infomation send a pull request
 ## Forking/Redistributing/Custom builds policy
 
 Do whatever you want, we don't care. Just follow the license. If you have any questions about this feel free to ask in an issue.
+
+All launcher code is available under the GPL-3 license.
```

Comment:
Do we want this to say launcher or PolyMC? Launcher is just the de-branded name.

Reply:
@ZekeSmith ^

Reply:
Bruh, you resolved it without addressing it.